src/client/src/apps/MainRoute/layouts/AppLayout.tsx

```diff
@@ -12,15 +12,11 @@ export interface Props {
 }
 
 const AppLayout: React.FC<Props> = ({ before, header, body, footer, after }) => {
-  // TODO - move to openfin-platform
-  //@ts-ignore
-  const isChildView = window.fin && window.fin.me && window.fin.me.isView
-
   return (
     <AppLayoutRoot data-qa="app-layout__root">
       {before}
 
-      {!isChildView && header !== null && <Header controls={header} />}
+      {header !== null && <Header controls={header} />}
```

Comment:
These changes are from a different PR so I am not sure what is going on here? You may need to rebase your changes on the latest ones from master?

Reply:
I'll create the new branch from master and only modify FooterVersion.tsx.